Marlin/planner.cpp

```diff
@@ -1303,8 +1321,12 @@ void Planner::set_position_mm(const AxisEnum axis, const float& v) {
 
 // Recalculate the steps/s^2 acceleration rates, based on the mm/s^2
 void Planner::reset_acceleration_rates() {
-  LOOP_XYZE(i)
+  uint32_t highest_acceleration_allaxes_steps_per_s2;
```

Comment:
1. Automatic (stack) variables are not initialized to 0.
2. This variable needs to be initialized to 1 for safety (since it is a divisor).